packages/@okta/vuepress-site/docs/guides/configure-signon-policy/prompt-factor-group/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Prompt for an MFA factor for a certain group
+---
+
+<RequireClassicUI/>
+
+The following are step-by-step instructions to configure an Okta Sign-On Policy to prompt a user for a factor (multifactor authentication (MFA)) when the user is a member of a certain group.
+
+### Create the policy container
+
+1. In the Admin Console, navigate to **Security** and then **Authentication**.
+
+2. Click the **Sign On** tab, and then click **Add New Okta Sign-on Policy**.
+
+3. In the **Add Policy** window, enter a **Policy Name** and a **Policy Description**.
+
+4. In the **Assign to Groups** box, enter the group name that you want to apply the policy to. In this example, we are specifying the **Contractor** group in our org. The group names must already exist before assigning them to a policy.
+
+5. Click **Create Policy and Add Rule**.
+
+### Create the rule
+
+1. In the **Add Rule** window, add a descriptivbe name for the rul in the **Rule name** box.
+
+2.  If there are any users in the **Contractor** group that you want to exclude from the rule, enter them in the **Exclude Users** box.
+
+3. For this use case example, leave the default of **Anywhere** in the **If User's IP is** drop-down box. Use this box to assign location parameters. This specifies what kind of location prompts authentication, for example, prompting a user for a factor when they aren't on the corporate network.
```

Comment:
This sentence seems out of place here:

> Use this box to assign location parameters. 

I'm not sure what this is intending to convey, but my guess is that it's something like this?

> Normally, this box is used to assign location parameters but in this example we are not adding any specific location parameters.